hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/service/AgentCoreEngine.java

```diff
@@ -291,15 +288,6 @@ public void startHawkularAgent(AgentCoreEngineConfiguration newConfiguration) {
             switch (this.configuration.getStorageAdapter().getType()) {
                 case HAWKULAR:
                     // if we are participating in a full Hawkular environment, we need to do some additional things:
-                    // 1. register our feed ID
-                    // 2. connect to the server's feed comm channel
-                    try {
-                        registerFeed(tenantIds);
```

Comment:
@jmazzitelli : I removed registerFeed as it doesn't have to be explicitly setup now. Except if you think that a feed should have an existence even if it has nothing associated with (no metric type, no resource type, no root resource)... in which case I should rollback this part.